CHANGELOG.md

```diff
@@ -51,7 +52,7 @@
     @override
     String toString() => 'Stuff{id: $id}';
   }
-
+  
```

Comment:
```suggestion

```

Reply:
I prefer a new-line after the class. I think it okay to correct small blemishes like this, even in the `CHANGELOG.md`